src/space.js

```diff
@@ -1,4 +1,4 @@
-module.exports = function Space(key, tags, reporters, errback) {
+function Space(key, tags, reporters, errback) {
```

Comment:
Why didn't you change Space to have an options object ?

Reply:
Makes sense.
Although I would still like to keep the `space` function with inline arguments - simply because it is less verbose and more convinient for reports:
```
metrics.space('my').space('test.space').increment(10)  
```
vs
```
metrics.space({ name:  'my' }).space({ name: 'test.space'}).increment({ value: 10})  
```

Reply:
yeah, the functions I think you should leave as they are now